.github/workflows/checks.yml

```diff
@@ -24,22 +27,31 @@ jobs:
         uses: jwlawson/actions-setup-cmake@4f73d30c2fc44a9466a3ed890fb8db7a7b554302
         with:
           cmake-version: '3.16.x'
-      - name: Set up Python 3.9.12
-        uses: actions/setup-python@v4
+      - name: Set up Python 3.9.12 and devnet
```

Comment:
This action only sets up python, it does not install devnet

Reply:
Because to run a setup-python job and install deps with caching those need to be in the same sub-task.

Reply:
Perhaps I don't understand something, but the way your change works is that you just removed step name, and did not combine two steps into one. So `Setup python 3.9.12 and devnet` is a single step that just sets up python, and `-run` is a separate, nameless step that installs devnet.

See -> https://github.com/software-mansion/starknet-jvm/actions/runs/5670681846/job/15389409119

I'd rather we leave the second step name as it was. Also, not sure if installing devnet takes this much time that it has to be cached, but if you want to proceed with this that's fine.